student.rb

```diff
@@ -1,7 +1,10 @@
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, age, name: 'unknown', parent_permission: true)
-    super(age, name, parent_permission,)
+    super(age, name, parent_permission)
     @classroom = classroom
+    classroom&.add_student(self)
   end
```

Comment:
- [ ] Great job so far. like the previous reviewer noted, there should be a clear method that serves as the 'belongs-to' side (a student belongs to a classroom).  something like

```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```